integration_tests/tests/iroha_cli.rs

```diff
@@ -112,3 +112,12 @@ async fn reads_client_toml_by_default() -> eyre::Result<()> {
 }
 
 // Add more CLI tests here!
+
+#[tokio::test]
+async fn get_server_version() -> eyre::Result<()> {
+    let network = NetworkBuilder::new().start().await?;
+    let client = network.client();
+    let version = client.get_server_version().unwrap();
```

Comment:
Using blocking client methods in async context can cause problems.

Please either use `spawn_blocking` (preferrable) or make the test non-async (cleaner).